Enable SSL by default

jnilau opened the pull request:
Enable SSL by default for both email report task versions

madkoo reviewed (commented):
@jnilau Everything looks fine. The only request I would ask is to update the readme-s for the tasks and state there that SSL is by default true.

jnilau commented:
> @jnilau Everything looks fine. The only request I would ask is to update the readme-s for the tasks and state there that SSL is by default true.

Thank you. I updated the READMEs and fixed the relative paths of the images while I was at it.
